viz_scripts/ebike_specific_metrics.ipynb

```diff
@@ -240,19 +200,21 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "labels_eb = data_eb.Replaced_mode.value_counts(dropna=True).keys().tolist()\n",
-    "values_eb = data_eb.Replaced_mode.value_counts(dropna=True).tolist()\n",
-    "plot_title=\"Number of trips for each replaced transport mode for eBike only\\n%s\" % quality_text\n",
-    "file_name ='ntrips_ebike_replaced_mode%s.png' % file_suffix\n",
-    "pie_chart_mode(plot_title,labels_eb,values_eb,file_name)"
+    "if study_type == 'program':\n",
```

Comment:
Ditto.

Reply:
Removed all "if program" statements from this notebook, since it will now exit if the config does not specify program.